pkg/kapp/cmd/core/deps_factory.go

```diff
@@ -107,3 +117,14 @@ func (f *DepsFactoryImpl) summarizeNodes(config *rest.Config) string {
 		return fmt.Sprintf("%s, %d+", oldestNode.Name, len(nodes.Items)-1)
 	}
 }
+
+func (f *DepsFactoryImpl) newWarningHandler() rest.WarningHandler {
+	if !f.Warnings {
+		return rest.NoWarnings{}
+	}
+	options := rest.WarningWriterOptions{
+		Deduplicate: true,
```

Comment:
This will print warning header in a yellowish colour.
```suggestion
		Deduplicate: true,
		Color: true,
```